src/js/components/PlacementConstraintsSchemaField.js

```diff
@@ -131,11 +133,22 @@ export default class PlacementConstraintsSchemaField extends Component {
         batch.reduce(combineReducers({ constraints: JSONReducer }))
       )
     );
+    const isRequired = fieldProps.required ? "*" : "";
 
     return (
-      <div>
+      <div className="extra-bottom">
         <FieldLabel>
```

Comment:
Is there a reason this is a field label? Field labels are for form components. But this is a heading.

Reply:
Good point!